com.protonvpn.ProtonVPN.metainfo.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This should be submitted upstream, all apps should install a metainfo file

Reply:
Most apps have to deal with this manually. Some apps just ignore the releases and one just patches in the last one here. Not ideal but works.